pep-0212.txt

```diff
@@ -5,163 +5,180 @@ Last-Modified: $Date$
 
-        def irange(sequence):
-            return zip(range(len(sequence)), sequence)
+    def irange(sequence):
+        return zip(range(len(sequence)), sequence)
 
-    These functions could be implemented either eagerly or lazily and
-    should be easy to extend in order to accept more than one sequence
-    argument.
+These functions could be implemented either eagerly or lazily and
+should be easy to extend in order to accept more than one sequence
+argument.
 
-    The use of these functions would simplify the idioms for looping
-    over the indices and over both elements and indices:
+The use of these functions would simplify the idioms for looping
+over the indices and over both elements and indices::
 
-        for i in indices(sequence):
-            # work with index i
+    for i in indices(sequence):
+        # work with index i
 
-        for i, e in irange(sequence):
-            # work with index i and element e
+    for i, e in irange(sequence):
+        # work with index i and element e
 
 
 Methods for sequence objects
+============================
 
-    This solution proposes the addition of 'indices', 'items'
-    and 'values' methods to sequences, which enable looping over
-    indices only, both indices and elements, and elements only
-    respectively.
+This solution proposes the addition of 'indices', 'items'
+and 'values' methods to sequences, which enable looping over
+indices only, both indices and elements, and elements only
+respectively.
 
-    This would immensely simplify the idioms for looping over indices
-    and for looping over both elements and indices:
+This would immensely simplify the idioms for looping over indices
+and for looping over both elements and indices::
 
-        for i in sequence.indices():
-            # work with index i
+    for i in sequence.indices():
+        # work with index i
 
-        for i, e in sequence.items():
-            # work with index i and element e
+    for i, e in sequence.items():
+        # work with index i and element e
 
-    Additionally it would allow to do looping over the elements
-    of sequences and dicitionaries in a consistent way:
+Additionally it would allow to do looping over the elements
+of sequences and dicitionaries in a consistent way::
 
-        for e in sequence_or_dict.values():
-            # do something with element e
+    for e in sequence_or_dict.values():
+        # do something with element e
 
 
 Implementations
+===============
 
-    For all three solutions some more or less rough patches exist
-    as patches at SourceForge:
+For all three solutions some more or less rough patches exist
+as patches at SourceForge:
 
-        'for i indexing a in l': exposing the for-loop counter[3]
-        add indices() and irange() to built-ins[4]
-        add items() method to listobject[5]
+- 'for i indexing a in l': exposing the for-loop counter [3]_
+- add ``indices()`` and ``irange()`` to built-ins [4]_
```

Comment:
I suggest using \`\`...\`\` formatting consistently (or don't use it at all) CONSTANTs and function()s aren't the only things that can be wrapped by \`\`...\`\`. If you change irange() to \`\`irange()\`\`, I'd expect the following sentence to be formatted too:

```
This solution adds two built-in functions 'indices' and 'irange'.
```

I guess you have a script that converts these PEPs to reST format, but It would be nice to do some proofreading before sending a PR.